developer_docs/release.md

```diff
@@ -65,7 +66,9 @@ ls target/helm/repo
 
 The Docker image not been pushed by the previous build step so must be done manually, e.g.
 ```
-docker push ibmcom/restconf-driver:0.0.1
+ echo <IAMAPIKEY> | docker login --username iamapikey --password-stdin icr.io/ibm/cp4na-drivers/
+ docker push icr.io/ibm/cp4na-drivers
+
 ```
 
 Complete the following:
```

Comment:
It looks like the release instructions are missing the automation steps and are just what would have been the fallback "Manual Approach".

Reply:
I've added in the missing steps.